src/dashboards/selectors/index.ts

```diff
@@ -28,12 +28,39 @@ export const getTimeRange = (state: AppState): TimeRange => {
   return state.ranges[contextID] || DEFAULT_TIME_RANGE
 }
 
+/**
+ *  is the string a valid UTC time without any time zone information?
+ *  ie:  "2021-07-17T14:00:00.000Z" is valid
+ *  if it has any timezone offset, then it is not a valid UTC time string
+ *
+ * @param str
+ */
+function isUtcTime(str) {
```

Comment:
what do you think of tweaking `str` (which is kind of obvious) to something like `formattedTime` or something similar to indicate that it's a string in the form of `2021-07-17T00:00:00.000Z`?  i know it's in the comment, but some people don't read those, and they get erased in sourcemaps